README.md

```diff
@@ -46,122 +44,150 @@ And iVSR is adaptable for deployment on a single device, a distributed system, c
+```bash
+source <workspace>/ivsr_ov/based_on_openvino_2022.3/openvino/install/setupvars.sh
+```
+
+For other OpenVINO versions installed via official packages, manual environment setup is not required.
+
+Once the build is successfully completed, refer to [section 3.2](#32-run-with-ffmpeg) for instructions on using the FFmpeg command line to run the pipelines. Feel free to modify and update these scripts as needed. For newly released OpenVINO versions, please follow the [manual build](#22-install-dependencies-and-build-ivsr-manually) guide.
+
+## 2.4 Install dependencies and build iVSR using Dockerfile
+To simplify the environment setup, Dockerfiles are provided. Follow the [Docker image build guide](docs/docker_image_build.md#docker-image-build-guide) to build the Docker image and run the application in Docker containers.
 
 # 3. How to use iVSR
-Both `vsr_sample` and FFmpeg integration are provided to run inference on the iVSR SDK. Execute the following commands to setup the env before executing them. <br>
+You can run inference on the iVSR SDK using either the `vsr_sample` or the `ffmpeg` integrated with the iVSR plugin. Before running them, set up the environment with the following commands:
 ```bash
 source <OpenVINO installation dir>/install/setupvars.sh
 export LD_LIBRARY_PATH=<Package dir>/ivsr_sdk/lib:<OpenCV installation folder>/install/lib:$LD_LIBRARY_PATH
 ```
-Please note that the current solution is of `pre-production` quality.<br>
-
+Note that the current solution is of `pre-production` quality.
 
 ## 3.1 Run with iVSR SDK sample
-`vsr_sample` has been developed using the iVSR SDK and OpenCV. For guidance on how to run inference with it, please refer to this [section](./ivsr_sdk/README.md#vsr-sample).<br>
+The `vsr_sample` is developed using the iVSR SDK and OpenCV. For detailed instructions on running inference with it, refer to this [section](./ivsr_sdk/README.md#vsr-sample).
 
 ## 3.2 Run with FFmpeg
-Once the FFmpeg plugin patches have been applied and FFmpeg has been built, you can refer to [the FFmpeg cmd line samples](ivsr_ffmpeg_plugin/README.md#how-to-run-inference-with-ffmpeg-plugin) for instructions on how to run inference with FFmpeg.<br>
+After applying the FFmpeg plugin patches and building FFmpeg, refer to [the FFmpeg command line samples](ivsr_ffmpeg_plugin/README.md#how-to-run-inference-with-ffmpeg-plugin) for instructions on running inference with FFmpeg.
 
 # 4. Model files
-Only models in OpenVINO IR format is supported by iVSR. Please reach out to your Intel representative to obtain the model files which are not included in the package.<br>
-
+iVSR supports only models in OpenVINO IR format. Contact your Intel representative to obtain the model files, as they are not included in the package.
```

Comment:
change "included in the package" to "included in this repo"?

Reply:
will change.